model/readonlydictionary.rst

```diff
@@ -0,0 +1,128 @@
+
+Using ``IReadOnlyDictionary`` with POCOs
```

Comment:
Is it POCOs or Poco's? In another PR you use Poco's. I think it is better to be uniform about this.

Reply:
Should be POCOs (just looked it up), I'll chase down the others.